pkg/gateway/gateway.go

```diff
@@ -1027,7 +1030,7 @@ func (n *jfsObjects) PutObjectPart(ctx context.Context, bucket, object, uploadID
 	var etag string
 	if err = n.putObject(ctx, bucket, p, r, opts, func(tmpName string) {
 		etag = r.MD5CurrentHexString()
-		if n.fs.SetXattr(mctx, tmpName, s3Etag, []byte(etag), 0) != 0 {
+		if n.gConf.KeepEtag && n.fs.SetXattr(mctx, tmpName, s3Etag, []byte(etag), 0) != 0 {
```

Comment:
I think we should keep this, and verify that in CompleteMultipartUpload

Reply:
OK, I will keep the code as is.